doc/Programming-guide/Using-Kaa-endpoint-SDKs/C++/SDK-Edison/index.md

```diff
@@ -81,6 +81,21 @@ Refer to [the official Edison getting started guide](https://software.intel.com/
    opkg install sqlite3
    ```
 
+7. Create .profile file and set LD_LIBRARY_PATH.
+
+   Add a profile file.
+   ```
+   nano .profile
+   ```
+   Write the following line into the added profile file and then save the file.
+   ```
+   export LD_LIBRARY_PATH=/usr/local/lib
+   ```
```

Comment:
note the user could already have a custom `LD_LIBRARY_PATH`, so we need to append here, not replace.